docs/api-proposal/proposed-routes.md

```diff
@@ -0,0 +1,76 @@
+This doc proposes a new set of API endpoints for the multi-jurisdiction audit flow.
+
+Some principles I tried to follow:
+
+- Reuse existing routes when available
+- Use REST conventions
+- Avoid reusing routes across roles. Instead of reusing one route and returning different scoped responses, make different roles query different endpoints with explicit scoping in the route. That way we just have to check if a user has access to the data they are asking for, as opposed to trying to filter data based on their access. (I learned this one the hard way.)
+
+New routes are in **bold**.
+
+# Audit Admin: Audit setup
+
+![](./create-audit.png)
+
+- POST /election/new - create new audit
+- **GET /election** - get all audits
```

Comment:
nit: `GET /election/` for listing a set of audits / elecitons.

Reply:
actually, thinking about this some more: this is one call that depends on authentication, and as per the principle outlined by @jonahkagan (which I agree with) that API calls should not return values that depend on authentication -- except one call --, we already have the list of audits that map to a user in `/auth/me`. How about we kill `GET /election` and just use `/auth/me` to get the elections that a user is mapped to. `/auth/me` is then the only auth-dependent API return value.